app/controllers/posts_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
   def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
+    @comments = @post.comments.includes(:author)
   end
 
```

Comment:
- [ ] Kindly add  _load_and_authorize_resource_ method to your post controller. It will use a before action to load the resource into an instance variable and authorize it for every action in the controller.
![image](https://user-images.githubusercontent.com/91486791/233420335-7966a5a1-b083-4d60-be50-89a7cab7ea6b.png)